plugins/UserCountry/VisitorGeolocator.php

```diff
@@ -305,6 +305,11 @@ private function getDefaultProvider()
     public static function getDefaultLocationCache()
     {
         if (self::$defaultLocationCache === null) {
+            if (!class_exists(Transient::class)) {
+                $ex = new \Exception();
+                print $ex->getTraceAsString()."\n";
+                exit;
+            }
```

Comment:
note to self (or whoever wants to take care of it): need to get rid of this debugging code